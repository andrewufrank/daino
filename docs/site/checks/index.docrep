DocRep {yam = Object (fromList [("fileEntries",Array []),("style",Null),("indexPage",Bool True),("link",String ""),("bibliography",Null),("lang",String "DLenglish"),("date",String "Jan. 4, 2019"),("indexSort",String "reverseDate"),("keywords",String "test"),("author",String "AUF"),("dirEntries",Array []),("abstract",String "The directory for experiments."),("title",String "primary index for Blog"),("fn",String "/home/frank/Workspace8/ssg/docs/site/dough/Blog/index.md"),("pageTemplate",String "page3.yaml"),("publish",Null)]), pan = Pandoc (Meta {unMeta = fromList []}) [Para [Str "an",Space,Str "index",Space,Str "page",Space,Str "for",Space,Str "Blog"]]}