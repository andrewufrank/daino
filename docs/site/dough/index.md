---
# values for the page individually
pageTemplate : page3
page-title : MyBlog
page-title-postfix : landingPage.md
#pagetitle : landingPage  # the same as title
#subtitle : subtitle text
#title-prefix : MyBlog
# author: auf   # only if different from site author
date: 2010-01-29
keywords: homepage
abstract: |
    The index page for all.

---

My landing page (TODO instead of a menu) 1 :

