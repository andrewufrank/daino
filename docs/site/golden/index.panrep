Panrep {panyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/index.md", dyLink = "index.md", dyLang = DLenglish, dyTitle = "XX's homepage", dyAbstract = "The index page for all.", dyAuthor = "FILL AUTHOR", dyDate = Just "2010-01-29", dyKeywords = "homepage", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/index.md", link = "index.md", title = "XX's homepage", abstract = "The index page for all.", author = "FILL AUTHOR", date = "2010-01-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, panpan = Pandoc (Meta {unMeta = fromList [("abstract",MetaBlocks [Para [Str "The",Space,Str "index",Space,Str "page",Space,Str "for",Space,Str "all."]]),("date",MetaInlines [Str "2010-01-29"]),("keywords",MetaInlines [Str "homepage"]),("page-title",MetaInlines [Str "MyBlog"]),("pageTemplate",MetaInlines [Str "page3"]),("title",MetaInlines [Str "XX's",Space,Str "homepage"])]}) [Para [Str "My",Space,Str "landing",Space,Str "page",Space,Str "(TODO",Space,Str "instead",Space,Str "of",Space,Str "a",Space,Str "menu)",Space,Str "1",Space,Str ":"]]}