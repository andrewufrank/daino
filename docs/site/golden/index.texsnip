TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/index.md", dyLink = "index.md", dyLang = DLenglish, dyTitle = "XX's homepage", dyAbstract = "The index page for all.", dyAuthor = "FILL AUTHOR", dyDate = Just "2010-01-29", dyKeywords = "homepage", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/index.md", link = "index.md", title = "XX's homepage", abstract = "The index page for all.", author = "FILL AUTHOR", date = "2010-01-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "My landing page (TODO instead of a menu) 1 :"}