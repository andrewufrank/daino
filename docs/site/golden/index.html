<!DOCTYPE html>
<!-- the master page for the pandoc templating mechanism
	needs page-title, page-title-postfix
		author date and keywords for indexing
    contentHtml, beforeContent, afterContent to wrap htmlcontent  
	uses style.css as default, other can be loaded with key css 
	-->
<html lang="DLenglish">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />


  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
	  </style>

	
	
          <meta name="author" content="FILL AUTHOR" />
              <meta name="dcterms.date" content="2010-01-29" />
              <meta name="keywords" content="homepage" />
    
	<title></title>
	
        <style>
            body {
            margin: 0;
            font-family: Arial, Helvetica, sans-serif;
            }

            .hero-image {
            background-image: url('');
            background-color: white;
            height: 200px;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
            }

            .hero-text {
            transform: translate(-50%, -50%);
            color: white;
            text-shadow: 2px 2px 8px black;
            font-weight: bold;
            vertical-align: top;
            }
        </style>
    </head>

    <body>
    <!-- AF - my template for pandoc page (from the default) page3.dtpl -->
        

    <section class="hero">
        <div class="hero-image">
            
                <h1></h1>
                <h3></h3>
        <div/>
    </section>
    <section class="header">

            <section class="menu">
		<ul>
            			</ul>
    </section>
    <section>
                <header>
        <h1 class="title">XX's homepage</h1>
        
                <p class="author">FILL AUTHOR</p>
                </header>
         
    </section>

    <section class=menu2>
		<ul>
            					<li>
				    <a href=index.HTML><em>XX's homepage </em></a>
				    <br>The index page for all.		     
				    <span class=tiny>FILL AUTHOR "2010-01-29" 
            index.HTML Nothing
                    </span>
                    
    			</li>
            				</ul>
    </section>

        
<!-- blog text content section -->
    
    abstractThe index page for all. 
    content-from-panrep-blocks<p>My landing page (TODO instead of a menu) 1 :</p>
    


        <p class=tiny>
        Produced with SGG  on  from  with master4.dtpl.
        </p>

            </body>


</html>
